src/scripts/app.ts

```diff
@@ -1018,11 +1018,7 @@ export class ComfyApp {
 
   private showMissingNodesError(missingNodeTypes: MissingNodeType[]) {
     if (useSettingStore().get('Comfy.Workflow.ShowMissingNodesWarning')) {
-      if (isCloud) {
-        useDialogService().showCloudLoadWorkflowWarning({ missingNodeTypes })
-      } else {
-        useDialogService().showLoadWorkflowWarning({ missingNodeTypes })
-      }
+      useDialogService().showCloudLoadWorkflowWarning({ missingNodeTypes })
```

Comment:
nit: Don't need Cloud anymore, right?
```suggestion
      useDialogService().showLoadWorkflowWarning({ missingNodeTypes })
```

Reply:
✅